app.js

```diff
@@ -108,10 +108,16 @@ Test this function by hand in the console to get it working, and when you think
 
 // Write your code here
 function multiplyArray(testArray){ //eslint-disable-line
-
+  var arrayProduct = 1; 
+  for(var i = 0; i < testArray.length; i++) {
+    arrayProduct = multiply(arrayProduct, testArray[i])[0];
+    console.log(arrayProduct);
+  }
+  var message = 'The numbers ' + testArray[0] + ',' + testArray[1] + ',' + testArray[2] + ' have a product of ' + arrayProduct + '.'; 
```

Comment:
Just so you know, you do not need to concatenate each element of `testArray` individually.
Just concatenating the array itself will accomplish the same thing in this case, i.e.
```javascript
'The numbers ' + testArray + ' have a product of ' + arrayProduct + '.'; 
```